src/JMS/Serializer/Metadata/Driver/YamlDriver.php

```diff
@@ -204,8 +204,8 @@ protected function loadMetadataFromFile(\ReflectionClass $class, $file)
 
         if (isset($config['handler_callbacks'])) {
             foreach ($config['handler_callbacks'] as $direction => $formats) {
+                $direction = GraphNavigator::parseDirection($direction);
```

Comment:
as this store a different thing, it would be even better to use a different variable name instead of overwriting the variable